modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -149,12 +150,11 @@ public function remove_from_cart_attributes( $url, $key ) {
 		$product = $item['data'];
 
 		$new_attributes = sprintf(
-			'href="%s" data-product_id="%s" data-product_sku="%s"',
-			esc_attr( $url ),
-			esc_attr( $product->get_id() ),
-			esc_attr( $product->get_sku() )
+			'" data-product_id="%s">',
+			esc_attr( $product->get_id() )
 		);
-		$url = str_replace( 'href=', $new_attributes, $url );
+
+		$url = str_replace( '">', $new_attributes, $url );
```

Comment:
This logic assumes the `data-product_id` is the last attribute. Perhaps not best to depend on that being the case